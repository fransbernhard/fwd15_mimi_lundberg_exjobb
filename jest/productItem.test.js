import React, { Component } from 'react';
import renderer from 'react-test-renderer';
import ProductItem from '../src/productItem';
import Modal from 'react-modal';

describe('components', () => {
  describe('<ProductItem/>', () => {
    it('renders correctly', () => {
      const tree = renderer.create(
        <ProductItem
          product={
            {id: 1, category: 'paint', name: 'clowd', type: 'matt emulsion', stocked: true, size: '100x130', thumbnail: '23-sm.png', previewImg: "23.png"}
          }
          key= {
            {id: 1}
          }
        />
      ).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
 
