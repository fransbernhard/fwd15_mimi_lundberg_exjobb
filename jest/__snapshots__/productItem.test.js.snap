// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <ProductItem/> renders correctly 1`] = `
<div
  className="product hvr-sink"
  onClick={[Function]}
>
  <noscript />
  23-sm.png
  <h3>
    clowd
  </h3>
  <p>
    matt emulsion
  </p>
  <p>
    100x130
     cm
  </p>
</div>
`;
